e2e/DeviceLocked.test.js

```diff
@@ -0,0 +1,35 @@
+const Utils = require('./Utils');
+const Android = require('./AndroidUtils');
+const testIDs = require('../playground/src/testIDs');
+const exec = require('shell-utils').exec;
+const _ = require('lodash');
+
+const { elementByLabel, elementById, sleep } = Utils;
+const IS_RELEASE = _.includes(process.argv, '--release');
+const KEY_CODE_R = 46;
+
+describe('application lifecycle test', () => {
+  beforeEach(async () => {
+    await device.relaunchApp();
+  });
+
+  test('launch from locked screen', async () => {
+    await device.terminateApp();
+    await Android.pressKeyCode('26')
+    await Android.pressKeyCode('26')
```

Comment:
Why are we invoking this twice?

Reply:
One is to lock the device and the other is to press it again to start the unlock process.